### DISCUSSION TITLE
Cloning git to local machine

### DISCUSSION THREAD

**Discussion by C-Doll** [cooperrc/computational-mechanics#151] [Q&A] [answered]
hello,

I have been trying to clone the forked repository to my local machine but am running into issues. I download Git because cmd didn't recognize the "git clone" command, then once that was sorted and I opened one of the files I cloned with Jupyter but the top bar didn't have a "run" option. (see pic)
![image](https://user-images.githubusercontent.com/121252243/210010399-51860f74-60c2-466c-990d-cb562c8eaff2.png)

I'm kind of at a loss now and could really just just some straightforward directions on how to get this working on my local machine because the UConn servers are unreliable at the moment.

**Comment by cooperrc** [comment, marked as answer]
Oh, that's a good point. 

You'll need [jupytext](https://jupytext.readthedocs.io/en/latest/)

You can install Jupytext with

`pip install jupytext`

    or 
`conda install jupytext -c conda-forge`

If you open a new terminal from Jupyter, you can run one of those commands there

**Reply by C-Doll** [reply]
Thank you very much that fixed it.

For anyone else reading I also needed to install a few libraries again through cmd

"pip install pandas"
"pip install matplotlib"

I am sure there are others that will also need installing.
-C.Doll

**Reply by seancari1001** [reply]
I tried installing jupytext and it didn't work at all, its still showing up as this. I also tried stopping and restarting the server.
![jupytext](https://user-images.githubusercontent.com/121534300/210117503-25b96e1b-efad-426a-9526-27582bdcda57.PNG)

**Reply by cooperrc** [reply]
This view is the `raw text` behind the notebooks we use (I use a special markdown file `<name>.md` that supports running code), 

If you're running [Jupyter Lab](https://blog.jupyter.org/jupyterlab-is-ready-for-users-5a6f039b8906) you might need to right-click and choose open as notebook, instead of "editor"
